### PR TITLE
[Admin] Design course show

### PR DESCRIPTION
**Pull request by NgocVu1994**

### CODE REVIEW

File: app/views/layouts/_admin_header.html.erb [hunk 1]
```diff
@@ -18,12 +18,12 @@
           </li>
           <li class="dropdown user user-menu">
             <a href="#" class="dropdown-toggle" data-toggle="dropdown">
-              <%= avatar_user_tag(current_user, "user-image") %>
+              <%= avatar_user_tag(current_user, "user-image", Settings.image_user_size) %>
```
Comment (truonglocbinh) [inline app/views/layouts/_admin_header.html.erb]: remove () ?

File: app/views/layouts/_admin_header.html.erb [hunk 2]
```diff
@@ -18,12 +18,12 @@
...
               <span class="hidden-xs"><%= current_user.name %></span>
             </a>
             <ul class="dropdown-menu">
               <li class="user-header">
-                <%= avatar_user_tag(current_user, "img-circle") %>
+                <%= avatar_user_tag(current_user, "img-circle", Settings.image_user_size) %>
```
Comment (truonglocbinh) [inline app/views/layouts/_admin_header.html.erb]: remove ()

File: app/views/layouts/_admin_header.html.erb [hunk 3]
```diff
@@ -46,7 +46,7 @@
     <section class="sidebar">
       <div class="user-panel">
         <div class="pull-left image">
-          <%= avatar_user_tag(current_user, "img-circle") %>
+          <%= avatar_user_tag(current_user, "img-circle", Settings.image_user_size) %>
```
Comment (truonglocbinh) [inline app/views/layouts/_admin_header.html.erb]: remove()

File: app/views/layouts/_user_header.html.erb [hunk 4]
```diff
@@ -18,7 +18,7 @@
             </li>
             <li class="dropdown user user-menu">
               <a href="#" class="dropdown-toggle" data-toggle="dropdown">
-                <%= avatar_user_tag(current_user, "user-image") %>
+                <%= avatar_user_tag(current_user, "user-image", Settings.image_user_size) %>
```
Comment (truonglocbinh) [inline app/views/layouts/_user_header.html.erb]: remove()

File: app/views/users/_info.html.erb [hunk 5]
```diff
@@ -1,6 +1,6 @@
 <div class="col-md-6 ">
   <div class="form-group">
-    <%= avatar_user_tag(@user, "img-circle") %>
+    <%= avatar_user_tag(@user, "img-circle", Settings.image_profile) %>
```
Comment (truonglocbinh) [inline app/views/users/_info.html.erb]: remove()

**Comment by thao33** [comment]
Ok.

**Pull request merged**